README.md

```diff
@@ -2,7 +2,191 @@
+plan to eventually work on something else.
+[The video "Metamath Proof Explorer (set.mm) contributions visualized with Gource through 2019-10-04"]( https://www.youtube.com/watch?v=XC1g8FmFcUU&list=PL1jSu6GGefBm7RBP0Id2Sa9uyVuyhioAC&index=4) shows set.mm's growth over time.
+
+The databases included, in (approximate) decreasing size, are:
+
+* "set.mm" aka Metamath Proof Explorer (MPE) -
+  uses classical logic and
+  Zermelo–Fraenkel set theory with the axiom of choice (ZFC).
+* "iset.mm" aka "Intuitionistic Logic Explorer" -
+  uses intuitionistic logic instead of classical logic,
+  e.g., it does not presume that the law of the implied middle is true
+  in all cases.
+* "nf.mm" aka "New Foundations Explorer" - constructs mathematics using
+  Quine's "New Foundations" (NF) set theory axioms, a direct derivative
+  of the "hierarchy of types" set theory originally presented in
+  Whitehead and Russell's *Principia Mathematica*.
+* "ql.mm" aka "Quantum Logic Explorer" - Starts from the orthomodular
+  lattice properties proved in the Hilbert Space Explorer and takes
+  you into quantum logic.
+* "hol.mm" aka "Higher-Order Logic Explorer" - Starts with HOL (also
+  called simple type theory) and derives equivalents to ZFC axioms,
+  connecting the two approaches.
+* "peano.mm" - Peano arithmetic.
+* "big-unifier.mm" - a unification and substitution test for
+  Metamath verifiers, where small input expressions blow up to thousands
+  of symbols.  It is a translation of William McCune's "big-unifier.in"
+  for the OTTER theorem prover.
+* "miu.mm"  - a simple formal system for use as a demonstration based
+  on work by Hofstadter.
+* "demo0.mm" - a simple formal system used as a demonstration in
+  Chapter 2 of the Metamath book.
+
+## How are they verified?
+
+Changes ("commits") to any database are first automatically verified
+before they are accepted, using GitHub actions.
+
+The set.mm and iset.mm databases are the ones primarily being updated.
+Every change to these databases are verified by the following
+different verifiers:
+
+* metamath.exe aka Cmetamath (the original C verifier by Norman Megill)
+* checkmm (a C++ verifier by Eric Schmidt)
+* smetamath-rs (smm3) (a Rust verifier by Stefan O'Rear)
+* mmj2 (a Java verifier by Mel L. O'Cat and Mario Carneiro)
+* mmverify.py (a Python verifier by Raph Levien)
+
+Note that these are different verifiers written in different programming
+languages by different people. In addition, the verification algorithm
+is intentionally simple (it fits in two pages in the Matamath book).
+This provides *extremely* high confidence that the proofs are correct.
+Most math proofs are not formally verified at all (that is, where a
+computer verification verifies every step). The rest are typically only
+verified by a single program (which might have an error in it).
+These multiple independent verifications provide very high confidence
+of their being correct.
```

Comment:
"their" should probably be "the proofs'" (it's more natural to say a proof is correct than to say a verification is correct, IMO)

Reply:
Agreed. I'll add a few clarifying words to bring the point home.